soc/nxp/lpc/soc.yml

```diff
@@ -0,0 +1,27 @@
+family:
+- name: lpc
```

Comment:
`nxp_lpc`

Reply:
we are staying with lpc and made the above string change to reflect the same.